web/src/main/resources/application.properties-EXAMPLE

```diff
@@ -7,5 +7,5 @@ topbraid.url=
 topbraid.username=
 topbraid.password=
 
-# for crosswalk e.g. http://data.mskcc.org/ontologies/api/concept/crosswalk?vocabularyId=ONCOTREE&conceptId=
+# for crosswalk e.g. http://data.mskcc.org/ontologies/api/concept/crosswalk?vocabularyId={vocabularyId}&conceptId={conceptId}&histologyCode={histologyCode}&siteCode={siteCode}
```

Comment:
parameterizing the URL for RestTemplate
I guess we are not really separating the parameters from the base URL, but that's ok.